### PR TITLE
Sub track component

### PR DESCRIPTION
**Pull request by amolskh**
Sub track component Changes (Separate Pull request)

### CODE REVIEW

File: components/SubTrackDetails/SubTrackDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require('./SubTrackDetails.scss')
+
+import React from 'react'
+
+const SubTrackDetails = (props) => {
```
Comment (vikasrohit) [inline components/SubTrackDetails/SubTrackDetails.jsx]: Please use destructured properties i.e `(props)` to` {name, code, description, tracks}`

File: components/Router/Router.jsx [hunk 2]
```diff
@@ -71,6 +72,8 @@ const Component = () => (
         <Route path="/NavbarExample" component={NavbarExample} />
 
         <Route path="/TCFooterExamples" component={TCFooterExamples} />
+
+	<Route path="/SubTrackDetailsExample" component={SubTrackDetailsExample} />
```
Comment (vikasrohit) [inline components/Router/Router.jsx]: Minor thing but to be consistent please use proper indention.

**Comment by amolskh** [comment]
Fixed review comments

**Comment by vikasrohit** [comment]
Looks Good. Last thing, can you please use classnames module to set class names because it is bit messy to have condition operations in { }

**Comment by amolskh** [comment]
Fixed

File: components/SubTrackDetails/SubTrackDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+require('./SubTrackDetails.scss')
+
+import React from 'react'
+import classNames from 'classnames'
+
+const SubTrackDetails = ({name,code,description,tracks}) => {
+  const developClass = classNames(
+    {'verticalLine develop': true},
+    {hidden: tracks.indexOf('develop') === -1 }
+  )
```
Comment (vikasrohit) [inline components/SubTrackDetails/SubTrackDetails.jsx]: Better syntax to use would be `classNames('verticalLine', 'develop', {hidden: tracks.indexOf('develop') === -1})`

**Comment by amolskh** [comment]
ok fixed

**Comment by vikasrohit** [comment]
Thanks @amolskh Looks Good. I will merge it soon, however, please don't delete the branch until challenge closes so that we can implemented final fixes(if required) in review phase.

**Pull request merged**